docs/contribute/DEV_DOCS.md

```diff
@@ -8,6 +8,8 @@ To be an effective, productive contributor, please start by reading the
 [**Nextstrain contributing guide**](https://github.com/nextstrain/.github/blob/master/CONTRIBUTING.md)
 for useful information about how to pick an issue, submit your contributions, and so on.
 
+Please see the "Source" tab in [Augur's installation instructions](https://docs.nextstrain.org/projects/augur/en/stable/installation/installation.html) for details on how to install Augur for development.
```

Comment:
Would be nice if we could make the source tab auto selected with an html anchor so that one knows exactly what to look at but I don't think that's possible with our current RTD setup unfortunately

Reply:
I've also thought of this occasionally but it's not possible at the moment. There's a feature request: https://github.com/executablebooks/sphinx-tabs/issues/160

Reply:
Yeah, I looked to see if there was some anchor I could link to, didn't see anything... 😒